flow_screen_components/LightningInputFSC/README.md

```diff
@@ -0,0 +1,126 @@
+# Lightning Input
+This is a port of the [Lightning Input](https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/aura_compref_lightning_input.htm) component, which includes the following input types:
+
+checkbox | checkbox-button | color | date | datetime-local | email | month | number | password | radio | range | tel | time | toggle | url | week
+
+## How It Works ##
+
+The component has two required attributs: `type` (possible values noted above) and `label`, which must be set in the Flow. There are also several type-specific attributes and validation attributes available ([see here for full documentation]((https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/aura_compref_lightning_input.htm)))
+
+A `value` attribute captures the user's input and can be used to set a Flow variable.
+
+For checkboxes and radio buttons, setting the "Group Name" attribute in the Flow will tie the buttons together.
+
+## Attributes and Examples by Type
+
+* type: "checkbox"
+   * label: "Red" 
+   * checked: "true"
+
+* type: "checkbox"
+   * label: "Blue" 
+
+* type: "checkbox"
+   * label: "Add pepperoni" 
+   * checked: "true" 
+   * value: "pepperoni" 
+
+* type: "checkbox-button" 
+  * label: "Add salami" 
+  * value: "salami" 
+
+* type: "color"
+   * label: "Color" 
+   * value: "#EEEEEE"
+
+* type: "date"
+   * label: "Birthday" 
+
+* type: "datetime-local" 
+  * label: "Birthday" 
+
+* type: "email"
+   * label: "Email" 
+   * value: "abc@domain.com" 
+
+* type: "month"
+   * label: "Birthday" 
+
+* type: "number"
+   * label: "Number" 
+   * value: "12345"
+
+* type: "number"
+   * label: "Enter a number" 
+
+* type: "number"
+   * label: "Enter a decimal value" 
+   * step: "0.001"
+
+* type: "number"
+   * label: "Enter a percentage value" 
+   * formatter: "percent" 
+   * step: "0.01" 
+
+* type: "number"
+   * label: "Enter a dollar amount" 
+   * formatter: "currency" 
+   * step: "0.01" 
+
+* type: "password"
+   * label: "Password" 
+
+* type: "radio"
```

Comment:
their implementation of radio here seems a little dysfunctional. first of all radio buttons have always been single select. Here, you could select several at once. But that's what checkboxes are for. And you have to write code to look at each once, check the value of checked, and take appropriate action. I can't think of a scenario where I'd prefer this to the built in radio button solution in Flow (which is not without its own downsides). Leave it in, I guess, but I'm inclined to deemphasize it...

Reply:
I was confused by the multiselect radios too. I've noticed multiselect radio buttons are used all over the on the SLDS site (e.g., the facets to the right of their component examples)--seems like a design choice being made there.

Reply:
Sigh. Kids these days.....